crossIBC.ps1

```diff
@@ -0,0 +1,62 @@
+param(
+    [Parameter(Mandatory=$true)][string]$ToolSource,
+    [Parameter(Mandatory=$true)][string]$ToolUser,
+    [Parameter(Mandatory=$true)][string]$ToolPAT,
+    [bool]$UsePartialNgen
+)
+
+function Find-IBCMerge {
+    $IBCMergePath = gci -recurse .\.packages | where {$_.Name.Contains("ibcmerge.exe")}
+    if(!$IBCMergePath)
+    {
+        $IBCMergePath = gci -recurse C:\microsoft.dotnet.ibcmerge | where {$_.Name.Contains("ibcmerge.exe")}
```

Comment:
Do you really need to search the C: drive?

Reply:
We should fix that bug rather than depend on it.  I hit a similar bug a couple days ago and fixed it: https://github.com/dotnet/corefx/commit/2e3b5eba7df595c7542d068be0d5b884b8c0a71a#diff-4b2f402501d23abbede6eac0f47783e4R70.  

Where's the restore optional tooling step implemented?  Sounds like its using the wrong property for the packages folder.

Reply:
ResolveOptionalTools is here: https://github.com/dotnet/buildtools/blob/0fa455c550abb4f49f4830a7f2fa78609f373fd0/src/Microsoft.DotNet.Build.Tasks/PackageFiles/OptionalTooling.targets

Reply:
Optional tools are restored here: https://github.com/dotnet/buildtools/blob/0fa455c550abb4f49f4830a7f2fa78609f373fd0/src/Microsoft.DotNet.Build.Tasks/PackageFiles/OptionalTooling.targets

Reply:
I did some investigation into this and was unable to figure out why it was restoring to the root. The package dir that was getting set was correct, but when it actually did the restore step it would not write them there.

Reply:
Hmm, when I try and repro this it is now restoring into the .packages directory. I will take out looking for the tool in the root.